packages/react-examples/src/experiments/FileTypeIcon/FileTypeIcon.Basic.Example.tsx

```diff
@@ -19,7 +19,7 @@ export class FileTypeIconBasicExample extends React.Component<{}, {}> {
         <h3>Size 48 csv icon as .png</h3>
         <Icon {...getFileTypeIconProps({ extension: 'csv', size: 48, imageFileType: 'png' })} />
         <h3>Size 64 model icon as .png</h3>
-        <Icon {...getFileTypeIconProps({ extension: 'blend', size: 64, imageFileType: 'png' })} />
```

Comment:
I seldom change the .example file but in this case I wanted to submit the change in order to show how the whiteboard file renders. This page is okay if we remove any of the extensions from it, because all it does is change what gets rendered in the Storyboard, and doesnt affect the capabilities of the package (at least that's my understanding but please call me out if there's a connection I'm not making).